src/lib/transition.ts

```diff
@@ -0,0 +1,24 @@
+export function showTitleWithMinimumScrolling() {
+  const headers = document.getElementsByTagName("h1")
+  if (headers.length) {
+    headers[0].scrollIntoView({
+      behavior: "smooth",
+      block: "nearest",
+      inline: "nearest",
+    })
+  }
+}
+
+export function getTitleFromRoute(route) {
+  let meta
+  for (let index = route.matched.length - 1; index >= 0; index -= 1) {
```

Comment:
Un `for..of` est souvent préférable car c'est plus lisible sur un tableau

Reply:
En fait, j'ai juste déplacé le code.
En plus, le tableau est lu à l'envers, je me dis que c'est plutôt ok.